easybuild/easyblocks/generic/pythonpackage.py

```diff
@@ -620,11 +647,17 @@ def make_module_req_guess(self):
     def make_module_extra(self, *args, **kwargs):
         """Add install path to PYTHONPATH"""
         txt = ''
-        self.set_pylibdirs()
-        for path in self.all_pylibdirs:
-            fullpath = os.path.join(self.installdir, path)
-            # only extend $PYTHONPATH with existing, non-empty directories
-            if os.path.exists(fullpath) and os.listdir(fullpath):
-                txt += self.module_generator.prepend_paths('PYTHONPATH', path)
+
+        # update $EBPYTHONPREFIXES rather than $PYTHONPATH
+        # if this Python package was installed for multiple Python versions
+        if self.multi_python:
+            txt += self.module_generator.prepend_paths(EBPYTHONPREFIXES, '')
```

Comment:
Doesn't it make sense to have the same checks than for the regular installation?

Reply:
Not really, because we're just adding the top-level installation directory to `$EBPYTHONPREFIXES`, not specific subdirectories like `lib/python*/site-packages`.

The `site-customize.py` script in framework determines which subdirectories are considered for the paths listed in `$EBPYTHONPREFIXES`

Reply:
Yes, but the top-level directory is useless if the directories inside do not exist. That's my point, if checking for the existence of those directories make sense.

Reply:
That's done via the `sanity_check_paths` default we inject?

Also, if those directories don't exist, the `import` check will fail anyway (unless those modules are available somewhere else for the right Python version, I guess)